### PR TITLE
Corrected the default value for name param in Nadam

### LINKED ISSUE #37179: Wrong "name" parameter default for Nadam optimizer
## URL(s) with the issue:
https://www.tensorflow.org/api_docs/python/tf/keras/optimizers/Nadam

## Description of issue (what needs changing):
Optional name attribute is said to default as "Adamax" rather than "Nadam"

### Clear description

name: Optional name for the operations created when applying gradients. Defaults to "Adamax".

### Correct links

https://github.com/tensorflow/tensorflow/blob/v2.1.0/tensorflow/python/keras/optimizer_v2/nadam.py#L33-L238

Source code correctly shows default as name="Nadam". Therefore documentation issue.

### Submit a pull request?

Not at the moment

### PR DESCRIPTION
**Pull request by ashutosh1919**
Fixes #37179 . @mihaimaruseac , Please review this one.

### CODE REVIEW

**Review by mihaimaruseac — COMMENTED** [review]
Would be great to merge multiple one line changes into a single PR to minimize hours of wasted CI.

**Comment by gbaned** [comment]
@ashutosh1919  Can you please check reviewer comments and keep us posted. Thanks!

**Comment by ashutosh1919** [comment]
@mihaimaruseac , Closing this one. As per your suggestion, I have added changes of this PR in #37192.
